main.js

```diff
@@ -34,37 +34,75 @@ var affirmationButton = document.querySelector('.affirmation');
 var mantraButton = document.querySelector('.mantra');
 var receiveMessageButton = document.querySelector('.receive-message-button');
 var clearMessageButton = document.querySelector('.clear-message-button');
+var addMessageButton = document.querySelector('.add-message-button')
+var submitButton = document.querySelector('.submit-button');
 var messageSection = document.querySelector('.message');
+var addAffirmationButton = document.querySelector('.affirmation-add');
+var addMantraButton = document.querySelector('.mantra-add');
 var bellImage = document.querySelector('svg');
+var addMessageForm = document.querySelector('#add-message-form');
+var messageInput = document.querySelector('.add-message-input');
+
 
 receiveMessageButton.addEventListener('click', randomize);
 clearMessageButton.addEventListener('click', clearMessage);
+addMessageButton.addEventListener('click', displayInput);
+submitButton.addEventListener('click', submitMessage);
 
-function findType() {
-  if (affirmationButton.checked) {
+function findType(affirmation, mantra) {
+  if (affirmation.checked) {
     return affirmations;
   }
-  if (mantraButton.checked) {
+  if (mantra.checked) {
     return mantras;
   }
 }
 function randomize() {
```

Comment:
Updated to pass in arguments since I am using function in two places now (receiving message and submitting a new message)